src/components/ComboButton/ComboButtonItem/ComboButtonItem.js

```diff
@@ -0,0 +1,14 @@
+/**
+ * @file Combo button item.
+ * @copyright IBM Security 2019
+ */
+
+import React from 'react';
+import { namespace } from '../ComboButton';
+
+const ComboButtonItem = props => {
```

Comment:
It's probably best to document the necessary prop types for this component otherwise consumers won't know what they need.